pycparser/c_generator.py

```diff
@@ -300,10 +300,12 @@ def _generate_struct_union_enum(self, n, name):
             body_function = self._generate_struct_union_body
         else:
             assert name == 'enum'
-            members = () if n.values is None else n.values.enumerators
+            members = None if n.values is None else n.values.enumerators
             body_function = self._generate_enum_body
         s = name + ' ' + (n.name or '')
         if members is not None:
+            # None means no members
```

Comment:
What does this comment refer to now, in the current revision of the code?

Reply:
Same thing as before.
- `None` still means that no curly-braces-enclosed members are present (handled by `p_struct_or_union_specifier_1` in the parser.
- Non-empty sequence means curly braces with members.
- Empty sequence means curly braces with no members.

Line 303 (translating `None` to an empty `tuple`) caused enums to always be suffixed with (possibly empty) curly braces.